### ISSUE TITLE
why Asymmetric sets for modifications in mass_alphabet.txt instead of Symmetric sets?

### ISSUE THREAD

**Issue by irleader** [snijderlab/stitch#254]
For modifications like Deamidation of N (same mass as D), it seems there is no difference whether N is from the temple, D is from the input reads or vice versa. 

e.g, Reference Template has N (could be deamidated), and de novo method predicts D, it is possible to be correct.
e.g, Reference Template has D , and de novo method predicts N(Deamidation) ,it is also correct.

Therefore can I use :

    Symmetric sets ->
        Score: 3
        Sets :>
            N,D
        <:
    <-

instead of the original? :

    Asymmetric sets ->
        Score: 3
        Sets :>
            - Template sequence -- results -> in read sequence - Type
            N->D -Deamidation
        <:
    <-

**Comment by douweschulte** [comment]
Yes that is a valid observation. It is indeed possible for these situations to both arise.
